pkg/support/dump_windows_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package support
+
+import (
+	"fmt"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/utils/exec"
+	exectesting "k8s.io/utils/exec/testing"
+
+	"antrea.io/antrea/pkg/agent/config"
+	aqtest "antrea.io/antrea/pkg/agent/querier/testing"
+)
+
+type testExec struct {
+	exectesting.FakeExec
+}
+
+type fakeCmd struct {
+	exectesting.FakeCmd
+}
+
+func (te *testExec) Command(cmd string, args ...string) exec.Cmd {
+	f := new(fakeCmd)
+	f.CombinedOutputScript = append(f.CombinedOutputScript, func() ([]byte, []byte, error) {
+		return []byte(fmt.Sprintf("%s %s", cmd, strings.Join(args, " "))), nil, nil
+	})
+	return f
+}
+
+func (c *fakeCmd) CombinedOutput() ([]byte, error) {
+	return []byte{}, nil
+}
```

Comment:
I think there is no need to declare new structs to use FakeExec. It's supposed to instantiate FakeExec and FakeCmd when using them.

Reply:
updated